ktor-utils/common/src/io/ktor/util/Attributes.kt

```diff
@@ -7,17 +7,43 @@ package io.ktor.util
 /**
  * Specifies a key for an attribute in [Attributes]
  * @param T is type of the value stored in the attribute
- * @param name is a name of the attribute for diagnostic purposes
+ * @param name is a name of the attribute for diagnostic purposes. Couldn't be blank
  */
 public open class AttributeKey<T>(public val name: String) {
```

Comment:
While you are here, can you please change the type of `<T>` to `<T : Any>`? `Attributes` class uses this restriction